cmd/cstor-repl-ctrl/app/command/commands.go

```diff
@@ -0,0 +1,83 @@
+package command
+
+import (
+	goflag "flag"
+
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openebs/maya/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdName = "cstorReplCtrl"
+	usage   = fmt.Sprintf("%s", cmdName)
+)
+
+// CstorReplCtrlOptions defines a type for the options of cstorReplCtrl.
+type CstorReplCtrlOptions struct {
+	KubeConfig string
+	Namespace  string
+}
+
+// AddKubeConfigFlag is used to add a config flag.
+func AddKubeConfigFlag(cmd *cobra.Command, value *string) {
+	cmd.Flags().StringVarP(value, "kubeconfig", "", *value,
+		"Path to a kube config. Only required if out-of-cluster.")
+}
+
+// AddNamespaceFlag is used to add a namespace flag.
+func AddNamespaceFlag(cmd *cobra.Command, value *string) {
+	cmd.Flags().StringVarP(value, "namespace", "n", *value,
+		"Namespace to deploy in. If no namespace is provided, POD_NAMESPACE env.var is used. Lastly, the 'default' namespace will be used as a last option.")
```

Comment:
To deploy what? I am little confused here. Will it deploy a cstor volume replica in a namespace? 

Can it be described in the cli help.